tests/test_jointcal_decam.py

```diff
@@ -105,6 +105,53 @@ def test_jointcalTask_2_visits_constrainedAstrometry_no_photometry(self):
 
         self._testJointcalTask(2, relative_error, self.dist_rms_absolute, pa1, metrics=metrics)
 
+    def setup_jointcalTask_2_visits_constrainedPhotometry_no_astrometry(self):
+        """Help keep the constrainedPhotometry tests consistent and make
+        the differences between them more obvious.
+        """
+        self.config = lsst.jointcal.jointcal.JointcalConfig()
+        self.config.astrometryRefObjLoader.retarget(LoadAstrometryNetObjectsTask)
+        self.config.photometryRefObjLoader.retarget(LoadAstrometryNetObjectsTask)
```

Comment:
Should these both be `LoadAstrometryNetObjectsTask`?

Reply:
Yup. The reference catalog loader is the `astrometry.net` ObjectsLoader. Nothing to do with whether it's used for photometry or astrometry.